test_main.py

```diff
@@ -10,6 +10,7 @@
 import requests
 import kcidb
 
+# flake8: noqa
```

Comment:
Why?

Reply:
I/O data lines were going out of its limits, that's why have to disable it here. Same as test_monitor.py

Reply:
You can check that [here](https://github.com/kernelci/kcidb/blob/f4307e781b53162e8d53c702d3acb4eb74474476/kcidb/test_monitor.py#L11)

Reply:
You can see that there's a comment there explaining why we're disabling it. Please copy it as well.

Reply:
Okay